custom_components/waste_collection_schedule/waste_collection_schedule/source/karlsruhe_de.py

```diff
@@ -33,11 +39,11 @@
     "Bioabfall": "mdi:leaf",
     "Papier": "mdi:package-variant",
     "Wertstoff": "mdi:recycle",
-    "Sperrmüllabholung": "mdi:wardrobe",
+    "Sperrmüllabholung": "mdi:wardrobe"
 }
 
 
-API_URL = "https://web6.karlsruhe.de/service/abfall/akal/akal.php"
+API_URL = "https://web6.karlsruhe.de/service/abfall/akal_2023/akal.php"
```

Comment:
```suggestion
API_URL = "https://web6.karlsruhe.de/service/abfall/akal/akal_2024.php"
```
Shouldn't we use the 2024 version, since they might be deprecate the 2023 soon, too.
The 2024 is also mentioned here https://tsk.karlsruhe.de/abfuhrkalender, the at the bottom of the page.

Reply:
Not really professional, that they do it like this, but it seems that they started "versioning" it every year.

Reply:
You are right. In the meantime, they have changed the URL again. The other URL still works, but we should use the new one.
And yes, it's not really professional that they started to create a "version" every year. Let's see what happens next.